library/build.gradle

```diff
@@ -24,7 +24,7 @@ android {
 
 dependencies {
     implementation libraries.appCompat
-    implementation libraries.googlePlayServices
+    api libraries.googlePlayServices
```

Comment:
This change should not be required, could you please revert back to `implementation`?

Reply:
In this way user cannot call LocationRequest.create() in LocationConfiguration.Builder#locationRequest(LocationRequest) in app, because googlePlayServices dependency is hidden.  
But yes, user can add googlePlayServices dependency in own app.
Return anyway to `implementation`?

Reply:
Yes, that is intentional. They only need to create `LocationRequest` if they want to use GooglePlayServices, which in that case they should have it included explicitly. Otherwise it is not necessary, making it `api` instead of `implementation` will prevent using this library without GooglePlayServices, which is not what we want.
 
So, please do revert it to `implementation`.